ui/raidboss/data/04-sb/raid/o2n.js

```diff
@@ -0,0 +1,411 @@
+'use strict';
+
+// O2N - Deltascape 2.0 Normal
+[{
+  zoneRegex: {
+    en: /^Deltascape \(V2\.0\)$/,
+  },
+  timelineFile: 'o2n.txt',
+  timelineTriggers: [
+    {
+      id: 'O2N Paranormal Wave',
+      regex: /Paranormal Wave/,
+      beforeSeconds: 5,
+      infoText: function(data) {
+        if (data.activeTank == data.me) {
+          return {
+            en: 'Tank cleave on YOU',
```

Comment:
What do you think about using response: Responses.tankCleave() here? It doesn't have the "activeTank" part of it, but it's close?

Reply:
I suppose that's true. I was trying to get around the lack of `matches` elements in timeline triggers for... something, but I can't think what it was now. (It was very clear when I wrote it, of course, so clear that a comment explaining was *completely* and laughably unnecessary.)